### PR TITLE
solve the issue Set meta prop theme-color #5847

### PR DESCRIPTION
**Pull request by Shu12388y**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/composables/head/useHead.ts [hunk 1]
```diff
@@ -7,13 +7,21 @@ export const useHead = ({ store }: { store: Store<any> }) => {
...
+      const themeColor = store.getters['configuration']?.currentTheme?.themeColor
       return {
         meta: [
           {
             name: 'generator',
             content: [getWebVersion(), getBackendVersion({ store })].filter(Boolean).join(', ')
+          },
+          themeColor && {
+            name: 'theme-color',
+            content: '#1b223d'
```
Comment (pascalwengerter) [inline packages/web-runtime/src/composables/head/useHead.ts:19]: Where do you chose this color from? 🤔

**Comment by pascalwengerter** [comment]
@Shu12388y nice to see someone tackling this! Have you looked into my comment in the original issue regarding CSS variables and the theme-color? The `web` project can be customized, and ideally this gets reflected in the theme-color as well ;)

File: packages/web-runtime/src/composables/head/useHead.ts (hunk 1 above)
Comment (Shu12388y) [reply packages/web-runtime/src/composables/head/useHead.ts:19]: i have taken from the website of the owncloud it is the color of nav bar. Is there is any problem I can change the theme color just give me the theme color I will add it

Comment (pascalwengerter) [reply packages/web-runtime/src/composables/head/useHead.ts:19]: @Shu12388y please see my other comment in the PR regarding the CSS variable

**Comment by kulmann** [comment]
Closing this as it's become stale.

**Pull request closed without merging**